### ISSUE TITLE
LSLForge Native Executable Problem

### ISSUE THREAD

**Issue by zauberparacelsus** [raysilent/lslforge#64]
When starting up Eclipse with LSLForge 0.1.9.8, I get the following error message:

![image](https://user-images.githubusercontent.com/548168/146073585-efbd98cf-3f7f-481d-9c13-f47cf4d09bbb.png)

When manually entering that URL into a web browser, the page does not seem to exist.

**Comment by raysilent** [comment]
Is this Linux?

**Comment by zauberparacelsus** [comment]
Yes, it is on 64bit Manjaro Linux, kernel 5.15.7, and using the latest eclipse on its repos, 2021-12 (4.22)

**Comment by zauberparacelsus** [comment]
Quick follow-up, I see the following warnings when Eclipse is run from a console:
```
WARNING: Using incubator modules: jdk.incubator.vector, jdk.incubator.foreign
chmod: cannot access '/home/zauber/LSLForge': No such file or directory
```
